src/model/test/GroundHeatExchangerVertical_GTest.cpp

```diff
@@ -211,3 +212,10 @@ TEST_F(ModelFixture, GroundHeatExchangerVertical_GFunctions) {
   EXPECT_EQ(100, gFunctions.size());
   EXPECT_THROW(testObject.addGFunction(1.0, 1.5), openstudio::Exception);
 }
+
+TEST_F(ModelFixture, GroundHeatExchangerVertical_Deprecated) {
+  VersionString thisVersion(openStudioVersion());
+  VersionString removeInVersion(3, 3, 0);
+  ASSERT_LT(thisVersion, removeInVersion)
+    << "Please go delete the OS_DEPRECATED functions in GroundHeatExchangerVertical class then delete this test";
+}
```

Comment:
Trying something new to make sure the deprecated functions are actually removed when time is due.